src/components/flyout/flyout.tsx

```diff
@@ -301,10 +286,19 @@ export const EuiFlyout = forwardRef(
         closeButtonProps?.className
       );
 
+      const closeButtonStyles = euiFlyoutCloseButtonStyles(euiTheme);
+
+      const closeButtonCSSStyles = [
+        closeButtonStyles.euiFlyout__closeButton,
+        closeButtonStyles[closeButtonPosition],
+        side === 'left' && closeButtonStyles.outsideLeft,
```

Comment:
see above comment https://github.com/elastic/eui/pull/6213/files#r970006504
```suggestion
        closeButtonStyles.euiFlyout__closeButton,
        closeButtonStyles[closeButtonPosition],
        closeButtonStyles[side],
```